openverse_catalog/dags/data_refresh/dag_factory.py

```diff
@@ -1,135 +1,144 @@
 """
 # Data Refresh DAG Factory
 This file generates our data refresh DAGs using a factory function.
-These DAGs initiate a data refresh for a given media type and awaits the
-success or failure of the refresh. Importantly, they are also configured to
-ensure that no two data refresh DAGs can run concurrently, as required by
-the server.
-
-A data refresh occurs on the data refresh server in the openverse-api project.
-This is a task which imports data from the upstream Catalog database into the
-API, copies contents to a new Elasticsearch index, and makes the index "live".
-This process is necessary to make new content added to the Catalog by our
-provider DAGs available on the frontend. You can read more in the [README](
+For the given media type these DAGs will first refresh the popularity data,
+then initiate a data refresh on the data refresh server and await the
+success or failure of that task.
+
+Popularity data for each media type is collated in a materialized view. Before
+initiating a data refresh, the DAG will first refresh the view in order to
+update popularity data for records that have been ingested since the last refresh.
+On the first run of the the month, the DAG will also refresh the underlying tables,
+including the percentile values and any new popularity metrics. The DAG can also
+be run with the `force_refresh_metrics` option to run this refresh after the first
+of the month.
+
+Once this step is complete, the data refresh can be initiated. A data refresh
+occurs on the data refresh server in the openverse-api project. This is a task
+which imports data from the upstream Catalog database into the API, copies contents
+to a new Elasticsearch index, and finally makes the index "live". This process is
+necessary to make new content added to the Catalog by our provider DAGs available
+on the frontend. You can read more in the [README](
```

Comment:
I think "available to the API" would be more accurate than "on the frontend" here.

Reply:
Good call!